app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comments < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
```

Comment:
OPTIONAL

- [ ] It was a requirement to use the `author_id` as the foreign key in all the other tables instead of `:user`. Using a new reference of the user in the other tables would be better.

You may choose to make this change in all your tables in your schema.